tests/postgres_tests/test_operations.py

```diff
@@ -459,6 +459,24 @@ def test_writer(self):
             "),",
         )
 
+    def test_reduce_create_remove(self):
+        self.assertOptimizesTo(
+            [
+                CreateCollation(
+                    "sample_collation",
+                    "und-u-ks-level2",
+                    provider="icu",
+                    deterministic=False,
+                ),
+                RemoveCollation(
+                    "sample_collation",
+                    # Different locale
+                    "de-u-ks-level1",
+                ),
```

Comment:
I assume that holds true because Postgres doesn't allow you to have collations with different locales sharing the same name? Kind of two difference indices with difference parameters that are included in this case to allow the operation to be reversible, [something we missed with `RemoveField`](https://code.djangoproject.com/ticket/26739).

Reply:
> I assume that holds true because Postgres doesn't allow you to have collations with different locales sharing the same name?

Yes, exactly.

> Kind of two difference indices with difference parameters that are included in this case to allow the operation to be reversible, something we missed with RemoveField.

Yes, `RemoveCollation` can already be reversible because it accepts all parameters.